### ISSUE TITLE
high acc in train and very low acc in test ,i use the prms in my pytorch code . 

### ISSUE THREAD

**Issue by vicdu** [bearpaw/PyraNet#6]
First, I really appreciate your work！
I want to write prms in pytorch.But the problem is ,loss and acc (badly)basically unchanged in the test set.Where did I go wrong?
thanks
